trlx/trlx.py

```diff
@@ -125,5 +125,8 @@ def train(  # noqa: C901
     )
     trainer.add_eval_pipeline(eval_pipeline)
 
+    if config.train.resume_from_checkpoint and os.path.exists(config.train.resume_from_checkpoint):
+        trainer.load(config.train.resume_from_checkpoint)
```

Comment:
@maxreciprocate Dumb question: Should the trainer be loaded before the first call to `make_experience`, otherwise, say for PPO, you get the initial rollouts from a random init model?

Reply:
@jon-tow Nice catch! I'll mirror the changes concerning that, which aman already made for the NeMo PR https://github.com/CarperAI/trlx/pull/472/files#diff-363d9b838f4a7ffb88cbfeb961ce5d7f9b52013e1363d31647a32469e576bfa3L105